fal/orm/anime.py

```diff
@@ -35,7 +35,10 @@ def __repr__(self) -> str:
     def add_anime_to_database(
         id: int, name: str, season: Season, session: Session
     ) -> None:
-        """ Adds new anime row to database if it doesn't already exist """
+        """
+        (DEPRECATED: use models.anime.create instead)
+        Adds new anime row to database if it doesn't already exist
+        """
         query = session.query(Anime).filter(Anime.id == id)
         anime = query.one_or_none()
```

Comment:
I wonder if we could add a warning for deprecations. We could do something like the below (inspired from [here](https://stackoverflow.com/questions/2536307/decorators-in-the-python-standard-lib-deprecated-specifically), [here](https://stackoverflow.com/questions/43013083/typing-decorator-with-parameters-in-mypy-with-typevar-yields-expected-uninhabite), and [here](https://github.com/python/mypy/pull/3113)):

```python
import warnings
import functools
from typing import Any, Callable, cast, TypeVar

FuncT = TypeVar('FuncT', bound=Callable[..., Any])

def deprecated(
    reason: str = "This function has been deprecated."
) -> Callable[[FuncT], FuncT]:
    """
    This decorator marks functions as deprecated and sends a warning.
    """
    def decorator(func: FuncT) -> FuncT:
        @functools.wraps(func)
        def wrapped(*args, **kwargs):
            warnings.warn(f"Call to deprecated function {func.__name__}: {reason}",
                        category=DeprecationWarning,
                        stacklevel=2)
            return func(*args, **kwargs)
        return cast(FuncT, wrapped)
    return cast(Callable[[FuncT], FuncT], decorator)

@deprecated("use models.anime.create instead")
def add_anime_to_database(
    id: int, name: str, season: Season, session: Session
) -> None:
    """
    (DEPRECATED: use models.anime.create instead)
    Adds new anime row to database if it doesn't already exist
    """
    query = session.query(Anime).filter(Anime.id == id)
    anime = query.one_or_none()
    if anime:
        print(f"{anime} already exists in database")
    else:
        anime = Anime(id=id, name=name, season_id=season.id)
        print(f"Adding {anime} to database")
        session.add(anime)
```
This isn't necessary at all though, so we could do it in a later PR.

Reply:
Actually from your first link there's even a deprecation package already. Maybe I'll check that out!

Reply:
Yeah I saw that, wasn't sure if we wanted to use it because it's not very popular and doesn't seem like we would get support if we needed it for some reason.

Reply:
true, it doesn't even look like it's annotated. i'll use your linked recipe